### PR TITLE
Deshopify ch.swisstopo.geologie-geologischer_atlas[*].metadata

### PR DESCRIPTION
**Pull request by b4l**
https://sys-s.dev.bgdi.ch/649862037652

### CODE REVIEW

**Comment by faselm** [comment]
A "l" is missing in the names;)
GeologischerAtasRasterMeta ==> GeologischerAtllllllllllllllasRasterMeta

Some tooltips values are all set to "-", maybe it's normal I don't know

Looks good otherwise

**Comment by b4l** [comment]
> A "l" is missing in the names;) GeologischerAtasRasterMeta ==> GeologischerAtllllllllllllllasRasterMeta

Thanks, fixed it.

> Some tooltips values are all set to "-", maybe it's normal I don't know

This has been resolved.

File: chsdi/models/vector/stopo.py [hunk 1]
```diff
@@ -617,12 +609,31 @@ class GeolGenKarteGGK200Meta(Base, ShopProductGroupClass, Vector):
...
+    id = Column('bgdi_id', Integer, primary_key=True)
+    nummer = Column('nummer', Integer)
+    name = Column('name', Unicode)
+    massstab = Column('massstab', Unicode)
+    ausgabejahr = Column('ausgabejahr', Integer)
+    autoren = Column('autoren', Unicode)
+    the_geom = Column(Geometry2D)
+
+
+class GeologischerAtasRasterMeta(Base, GeologischerAtlasMeta, Vector):
```
Comment (faselm) [inline chsdi/models/vector/stopo.py:625]: 'l' still missing here

File: chsdi/models/vector/stopo.py [hunk 2]
```diff
@@ -617,12 +609,31 @@ class GeolGenKarteGGK200Meta(Base, ShopProductGroupClass, Vector):
...
+    __tablename__ = 'geologischer_atlas_raster_meta'
     __bodId__ = 'ch.swisstopo.geologie-geologischer_atlas.metadata'
 
-register('ch.swisstopo.geologie-geologischer_atlas.metadata', Ga25Meta)
+register('ch.swisstopo.geologie-geologischer_atlas.metadata', GeologischerAtasRasterMeta)
```
Comment (faselm) [inline chsdi/models/vector/stopo.py:629]: 'l' still missing here

File: chsdi/models/vector/stopo.py [hunk 3]
```diff
@@ -617,12 +609,31 @@ class GeolGenKarteGGK200Meta(Base, ShopProductGroupClass, Vector):
...
+
+
+class GeologischerAtasVectorMeta(Base, GeologischerAtlasMeta, Vector):
```
Comment (faselm) [inline chsdi/models/vector/stopo.py:632]: 'l' still missing here

File: chsdi/models/vector/stopo.py [hunk 4]
```diff
@@ -617,12 +609,31 @@ class GeolGenKarteGGK200Meta(Base, ShopProductGroupClass, Vector):
...
+    __tablename__ = 'geologischer_atlas_vector_meta'
+    __bodId__ = 'ch.swisstopo.geologie-geologischer_atlas_vector.metadata'
+
+register('ch.swisstopo.geologie-geologischer_atlas_vector.metadata', GeologischerAtasVectorMeta)
```
Comment (faselm) [inline chsdi/models/vector/stopo.py:636]: 'l' still missing here

**Review by faselm — APPROVED** [review]

**Review by ltclm — APPROVED** [review]
thank you!

please also remove deprecated views after the deploy:

- view_gridstand_gav25

- view_ga25_grid_shop

**Pull request merged**